deploy/chart/templates/0000_50_olm_09-aggregated.clusterrole.yaml

```diff
@@ -11,7 +11,7 @@ rules:
   resources: ["subscriptions"]
   verbs: ["create", "update", "patch", "delete"]
 - apiGroups: ["operators.coreos.com"]
-  resources: ["clusterserviceversions", "catalogsources", "installplans", "subscriptions"]
+  resources: ["clusterserviceversions", "catalogsources", "installplans", "subscriptions", "operators"]
```

Comment:
This makes sense -- @njhale would know more here -- should these changes be made here or in another place.

Reply:
@njhale Ping. Thanks